Formula/mmseqs2.rb

```diff
@@ -0,0 +1,71 @@
+class Mmseqs2 < Formula
+  desc "Software suite for very fast protein sequence search and clustering"
+  homepage "https://mmseqs.org"
+  # tag "bioinformatics"
+  # doi "10.1038/nbt.3988"
+  url "https://github.com/soedinglab/MMseqs2.git", :using => :git, :tag => "1-c7a89", :revision => "c7a89b6aa6d94796783e94d49750b3e0882573aa"
+  version "1-c7a89"
+  head do
+    url "https://github.com/soedinglab/MMseqs2.git", :using => :git
+    resource "documentation" do
+      url "https://github.com/soedinglab/MMseqs2.wiki.git", :using => :git
+    end
+  end
+
```

Comment:
You also need: `bottle :disable, "Requires SSE4.1 support"`

Reply:
Is it necessary to disable bottle building? The compiler does not need to have SSE support.